### DISCUSSION TITLE
Processor Technology VDM Implementation

### DISCUSSION THREAD

**Discussion by Gmanxxx** [thehighnibble/thehighnibble.github.io#128] [IMSAI8080esp replica]
Any opportunity to have a PT VDM implementation?  That would allow more CPM games including "Trek 80".

**Comment by udo-munk** [comment]
The VDM also is a memory mapped video device, and with that thing mapped usable memory is reduced to 48 KB :-(
It is inplemented in z80pack Altair 8800 emulation, you can have a look there.
